Jupyter_notebook/tutorial.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PIA2IaqUOeOA"
+      },
+      "source": [
+        "# Welcome to the NoisePy Colab Tutorial!\n",
+        "\n",
+        "This tutorial will walk you through the basic steps of using NoisePy to compute ambient noise cross correlation functions.\n",
```

Comment:
Please take a look at the latest version of the [colab notebook](https://colab.research.google.com/drive/1ir-p05dW34cj4biEZYqa7QXgXpjqv3fW#scrollTo=rR860zaZPZGH). The installation is now done through:
```
!pip install noisepy-seis
```
So there's no need to clone the repo, install requirements or cd to `src`.